english/dog-eat/dog-eat.md

```diff
@@ -0,0 +1,18 @@
+You notice that the walls are beginning to melt, but fortunately
```

Comment:
There is no way to pull or clone part of a repository, you get it as one big unit. Fortunately the whole thing is only a couple KB, which isn't *that* big as far as things go (compare to [this repo](https://github.com/torvalds/linux) which is many GB).

Working in large file trees  and finding the things you need is a good skill to practice. Good luck!